public/styles/app.css

```diff
@@ -0,0 +1,140 @@
+.page-header {
```

Comment:
Now that we moved `../build` into the top of the app, do we still need to check in all this `public` stuff?

Reply:
For the login page we do need it .. the imported styles are being loaded as style tags in the html.. the requests to public will only send something for login I believe

Reply:
I think i'd rather we didn't have a second separate copy of the CSS kicking around, but i'm not particularly bothered either way.

The approach you started taking with reading the manifest to render the HTML and use the webpack built assets would acheive this.